client/a8c-for-agencies/components/offering/style.scss

```diff
@@ -30,24 +30,39 @@
 			.a4a-offering-item__title {
 				color: var(--color-neutral-100);
 				font-weight: 500;
-				font-size: 1rem;
+				font-size: rem(16px);
+				line-height: 20px;
```

Comment:
See comment above

Reply:
Fixed in https://github.com/Automattic/wp-calypso/commit/0c52812e8e9f17f2d39a7250050c2d10c14ac488